services/core-api/app/extensions.py

```diff
@@ -76,7 +76,7 @@ def getJwtManager():
         raise AuthError({'code': 'auth_fail',
                     'description': 'Token issuer oidc.gov.bc.ca is no longer supported. Please contact the mds team.'}, 401)
 
-    if iss in test_config.JWT_OIDC_TEST_ISSUER:
+    if test_config.JWT_OIDC_TEST_ISSUER is not None:
```

Comment:
Think we also might need the `iss in test_config.JWT_OIDC_TEST_ISSUER` check here😅
Think it's the case locally that `JWT_OIDC_TEST_ISSUER` is set but the users token is actually issued by the `mds-public` issuer